README.md

```diff
@@ -65,22 +65,21 @@ import java.util.List;
 
 public class PageScreenshot {
   public static void main(String[] args) throws Exception {
-    Playwright playwright = Playwright.create();
-    List<BrowserType> browserTypes = Arrays.asList(
+    try (Playwright playwright = Playwright.create()) {
```

Comment:
Let's also update [source files](https://github.com/microsoft/playwright-java/tree/master/examples/src/main/java/org/example) in the examples project.